docs/sat_bootprep.md

```diff
@@ -543,9 +475,10 @@ Default bootprep input files are delivered by the HPC CSM Software Recipe
 product. You can access these files by cloning the `hpc-csm-software-recipe`
 repository.
 
-To do this, follow steps 1-7 of the procedure in [Viewing HPC CSM Software Recipe
-Variables](#viewing-hpc-csm-software-recipe-variables). Then, access the files in the
-`bootprep` directory of that repository:
+To do this, follow steps 1-7 of the procedure described in the **Accessing `sat
+bootprep` files** section of the [*Cray System Management
+Documentation*](https://cray-hpe.github.io/docs-csm/). The default input files
+can be found in the `bootprep` directory of the cloned repository:
```

Comment:
I would consider just condensing this paragraph into the previous one by adding "as described in the ... section of the Cray System Management Documentation." I don't feel like saying the steps is necessary here either as the number of steps could change, and then we'd be out of date here.